Tools/plutil/CMakeLists.txt

```diff
@@ -3,11 +3,8 @@ add_executable(plutil
   main.swift)
 target_link_libraries(plutil PRIVATE
   Foundation)
-# TODO(compnerd) properly propogate `BUILD_RPATH` to the target using CMake
-if(NOT CMAKE_SYSTEM_NAME STREQUAL Windows AND NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
-  target_link_options(plutil PRIVATE
-    "SHELL:-Xlinker -rpath -Xlinker $$$$ORIGIN/../lib/swift/$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>:$<TARGET_FILE_DIR:Foundation>")
-endif()
+set_target_properties(plutil PROPERTIES
+  INSTALL_RPATH "$ORIGIN/../lib/swift/$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>")
```

Comment:
Hey @compnerd, I just noticed that this broke the `plutil` command on linux, because it's not adding this relative rpath for 5.2 or master. I tried various ways to fix it, such as reverting back to
```
target_link_options(plutil PRIVATE "SHELL:-Xlinker -rpath -Xlinker \\\$\$ORIGIN/../lib/swift/$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>")
```
but nothing worked for me, maybe you know how to fix this.

Reply:
@CodaFi, I don't know how close you guys are to shipping the 5.2 release, but I get the following when running `plutil` from the latest official 5.2 snapshot build on linux, because of this missing rpath:
```
> ./swift-5.2-DEVELOPMENT-SNAPSHOT-2020-03-19-a-ubuntu18.04/usr/bin/plutil -help
./swift-5.2-DEVELOPMENT-SNAPSHOT-2020-03-19-a-ubuntu18.04/usr/bin/plutil: error while loading shared libraries: libicuucswift.so.65: cannot open shared object file: No such file or directory
> readelf -d swift-5.2-DEVELOPMENT-SNAPSHOT-2020-03-19-a-ubuntu18.04/usr/bin/plutil  | ag runpath
 0x000000000000001d (RUNPATH)            Library runpath: [/home/buildnode/jenkins/workspace/oss-swift-5.2-package-linux-ubuntu-18_04/build/buildbot_linux/swift-linux-x86_64/lib/swift/linux] 
```
Prefixing with `LD_LIBRARY_PATH` gets it working again, so it's clearly the rpath issue. I don't know how much this executable matters, but it's broken.